cmd/kubeadm/app/phases/upgrade/policy_test.go

```diff
@@ -44,9 +44,9 @@ func TestEnforceVersionPolicies(t *testing.T) {
 		{
 			name: "major upgrade",
 			vg: &fakeVersionGetter{
-				clusterVersion: constants.MinimumControlPlaneVersion.WithPatch(3).String(),
```

Comment:
I need to revert some of this changes because bump `CurrentKubernetesVersion` to `1.14.0` broke this test. We probably should use something like `constants.CurrentKubernetesVersion.WitchMinorMinus(1).WithPatch(3).String()` in  future.